src/main/webapp/WEB-INF/tags/shared/feedbackResponseCommentRowForFeedbackParticipant.tag

```diff
@@ -5,13 +5,12 @@
 <%@ taglib tagdir="/WEB-INF/tags/shared" prefix="shared" %>
 <%@ tag import="teammates.common.util.Const" %>
 <%@ attribute name="frc" type="teammates.ui.template.FeedbackResponseCommentRow" required="true" %>
-<%@ attribute name="responseIndex" %>
-<%@ attribute name="qnIndex" %>
-<%@ attribute name="isSessionOpenForSubmission" type="java.lang.Boolean"%>
+<%@ attribute name="responseIndex" type="java.lang.Integer" %>
+<%@ attribute name="qnIndex" type="java.lang.Integer" %>
```

Comment:
Will that result in duplicated ID?

Reply:
Yes, but only in instructor results, student records and student results where we don't use id of this Feedback Participant Comment Row. It will not result in duplicated id in submission edit page.

Reply:
> Yes, but only in instructor results

That's doesn't sound good to me.

Reply:
ID should be unique in a whole HTML page (see HTML5 specification).

We have many violation #8941 and I think it is not good idea to introduce new violations.

Reply:
Okay, what should be the id of feedback participant comments in instructor results page. Because currently it is `-qnindex-responseindex` and it wouldn't be unique in results page. Would adding `recipientindex` and  `giverindex` to `feedbackResponseCommentRowForFeedbackParticipant.tag` be okay?

Reply:
@xpdavid Can we not use this `responseindex` = `giverindex-recipientindex`?

Reply:
I think we might just leave it there and record it in #8941.

We will have a big UI framework updates in the incoming year and this issue won't be an "issue" if that is done.

Reply:
I tried your previous suggestion and it doesn't complicate things. Should I push it or leave it as it is?

Reply:
> I tried your previous suggestion and it doesn't complicate things. Should I push it or leave it as it is?

Good work 👍 . You can push it.